flow360/component/flow360_params/solvers.py

```diff
@@ -510,6 +515,8 @@ class TransitionModelSolver(GenericFlowSolverSettings):
         )
 
 
+TransitionModelSolverType = Union[NoneSolver, TransitionModelSolver]
+
```

Comment:
do we really need NoneSolver for Transition? I think preference here is just to omit this parameter.

Reply:
Sure. will remove it.